app/Architecture/ExceptionHandler.php

```diff
@@ -21,7 +21,15 @@ public function render($request, Throwable $e): Response
     {
         $response = parent::render($request, $e);
 
-        if (app()->environment("production") && in_array($response->status(), [500, 503, 429, 419, 404, 403, 401], true)) {
+        if (!app()->environment("production")) {
+            return $response;
+        }
+
+        if ($response->status() === 405) {
```

Comment:
Could you use `Response::METHOD_NOT_ALLOWED` and other consts instead of magic numbers?